.github/workflows/react-components.yml

```diff
@@ -29,8 +30,10 @@ jobs:
         working-directory: .
       - name: build
         run: npm run build:storybook && npm run build
+      - name: Setup kernel for Karma tests, increase watchers
+        run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
```

Comment:
is this really needed? it should just run once and finish in ci so it shouldn't need to watch for changes.

Reply:
No, looks like we don't, this is a leftover same as  --max-old-space-size. It was fixed when we solved the material-ui bundle problem